dcicutils/env_utils.py

```diff
@@ -56,6 +59,25 @@ def _orchestrated_app_case(orchestrated_app: OrchestratedApp, if_cgap, if_fourfr
 # CGAP_ENV_WEBPROD2_NEW is meaningless here. See CGAP_ENV_STAGING_NEW.
 CGAP_ENV_WOLF_NEW = 'cgap-wolf'  # Maybe not used
 
+_ALL_BEANSTALK_NAMES = [
+    # You can peek at contents of this variable to see how is_beanstalk_env will work,
+    # but DO NOT reference this directly from code outside of th is module. -kmp 18-Apr-2022
```

Comment:
extra space in comment ("... of th is module.")

Reply:
fixed